Dockerfile

```diff
@@ -1,11 +1,21 @@
-FROM node:12
+# GENERIC DOCKER SETUP
+FROM node:16
 WORKDIR /opt/jupiterone
 RUN apt-get update
-RUN wget https://github.com/CycloneDX/cyclonedx-cli/releases/download/v0.14.0/cyclonedx-linux-x64 -O /bin/cyclonedx
+
+# CYCLONEDX
+# Keep an eye out for releases of cyclonedx
+# We might need to bump a version when necessary
+RUN wget https://github.com/CycloneDX/cyclonedx-cli/releases/download/v0.19.0/cyclonedx-linux-x64 -O /bin/cyclonedx
 RUN chmod +x /bin/cyclonedx
+
 COPY . .
-RUN yarn install --production
+
+# NPM PACKAGES
 RUN npm install -g @cyclonedx/bom
```

Comment:
I realize this is old code, but wondering if we should install a specific version/range globally instead of the _latest_ by default? Seems dangerous to always install the latest. The package may introduce breaking changes across versions.

Something like:

```
RUN npm install -g @cyclonedx/bom@^3
```